src/vue/common/assets/AssetsList.vue

```diff
@@ -2,11 +2,22 @@
   <div class="assets-list">
     <drawer :is-shown.sync="isDetailsDrawerShown">
       <template slot="heading">
-        {{ 'asset-details.title' | globalize }}
+        <template v-if="isUpdateMode">
+          {{ 'asset-form.update-token-title' | globalize }}
+        </template>
+        <template v-else>
+          {{ 'asset-details.title' | globalize }}
+        </template>
       </template>
+      <asset-form
+        v-if="isUpdateMode"
+        :asset-for-update="selectedAsset.code"
+      />
       <asset-details
+        v-else
         :asset="selectedAsset"
-        @balance-added="updateSelectedAsset"
+        @balance-added="refreshSelectedAsset"
+        @update="isUpdateMode = true"
```

Comment:
At first look, I though `update` event tells us that asset is already updated.

Use `update-click`, `update-ask` or something like this. the `update` on it's own is quite confusing, because it gives us no information about what exactly was updated

Reply:
Renamed to `update-ask`